packages/builder/src/pages/builder/app/[application]/settings/oauth2/AddButton.svelte

```diff
@@ -86,22 +86,15 @@
 <Modal bind:this={modal}>
   <ModalContent onConfirm={saveOAuth2Config} size="M">
     <Heading size="S">Create new OAuth2 connection</Heading>
-
     <Body size="S">
       The OAuth 2 authentication below uses the Client Credentials (machine to
       machine) grant type.
     </Body>
     <Divider noGrid noMargin />
-    <Input
-      label="Name*"
-      placeholder="Type here..."
-      bind:value={config.name}
-      error={errors.name}
-    />
+    <Input error={errors.name} />
```

Comment:
The name and service URL are not bound to anything so the save function currently fails validation. This is complete [here](https://github.com/Budibase/budibase/pull/15753). I'm guessing everything will sink up after the merge. I couldn't verify that I could select an OAuth2 source.